### ISSUE TITLE
Python Version

### ISSUE THREAD

**Issue by loretoparisi** [awslabs/amazon-sqs-java-temporary-queues-client#20]
I did not find any Python implementation of the `AmazonSQSTemporaryQueuesClient`. Can you help?
Thank you.

**Comment by robin-aws** [comment]
Hi there! A python version of this library is on the backlog, but there's no ETA as of yet.

**Comment by loretoparisi** [comment]
@robin-aws thank you Robin.

**Comment by nirangad** [comment]
@robin-aws Same for Ruby as well? Thank you

**Comment by robin-aws** [comment]
@nirangad Yes, roughly the same status. Anecdotally I've heard more requests for Python though.

I'd encourage you to cut a separate issue for Ruby so we can tally +1's independently.

**Comment by nirangad** [comment]
@robin-aws Thanks a lot for the reply. I will create a separate issue :)

**Comment by paranthaman-v** [comment]
Any update on Python support

**Comment by rabinnh** [comment]
We just implemented a data pipeline that depends on SQS receiving a success/failure SQS message, but we need for multiple clients to be able get the messages intended for them.  We planned on using virtual queues after reading the docs, and then found no Python support.  So please add us to the list of requests for Python.

**Comment by hankehly** [comment]
We would really like to use a python implementation too.

**Comment by jptavarez** [comment]
I've developed a Python SQS Client that also supports Two-way Messaging (request/response):

https://github.com/jptavarez/python-sqs-client

**Comment by nathanielram** [comment]
Any update on python support?

**Comment by sidarth164** [comment]
Any update on python support?

**Comment by osalloum** [comment]
+1

**Comment by sharonvardi** [comment]
+1

**Comment by mikedavidson-evertz** [comment]
+1

**Comment by aviad-dev** [comment]
👍🏿

**Comment by the-rock** [comment]
+1 python

**Comment by danibachar** [comment]
+1 python
